builder.go

```diff
@@ -85,7 +85,13 @@ func (b Builder) Build(ctx context.Context, outputFile string) error {
 	env = setEnv(env, "GOOS="+b.OS)
 	env = setEnv(env, "GOARCH="+b.Arch)
 	env = setEnv(env, "GOARM="+b.ARM)
-	if b.RaceDetector && !b.Compile.Cgo {
+
+	raceArg := "-race"
+
+	// trim debug symbols by default
+	buildFlags := b.osEnvOrDefaultValue("XK6_BUILD_FLAGS", "-ldflags=-w -s")
```

Comment:
I am not certain if we don't want to not have this as the default :thinking:

Reply:
But maybe leave that for a separate PR later as I guess we will need to discuss it with the team more :shrug: . I personally don't think the extra space is a problem, but maybe there are other reasons to trim them by default